app/controllers/concerns/response_error_handler.rb

```diff
@@ -0,0 +1,9 @@
+module ResponseErrorHandler
+  extend ActiveSupport::Concern
+
```

Comment:
just wondering 

should this snippet sit here? 
```ruby
        rescue_from Pagy::OverflowError do |_exception|
          render_json_error(code: 400, message: I18n.t("exceptions.pagy.overflow"), status: :bad_request)
        end
```

Reply:
I wondered that too. I'm not too fussed but I did think that putting it into the module might be a better place for it (and others).

Reply:
Actually, I think we should handle these rescues in the relevant controllers to avoid polluting this module with exceptions which might not make sense for other controllers.

Reply:
We should move that block Sau quoted into a separate, pagy overflow handling concern and include it.